### DISCUSSION TITLE
TUNNEL CONNECTION FAILED - Help i keep getting this chrome closes everytime i enable a instance

### DISCUSSION THREAD

**Discussion by ChickenFlavor** [KevinBytesTheDust/CVAmp#133] [Q&A] [answered]
![IMG_20230525_025332_768](https://github.com/jlplenio/crude-twitch-viewer-bot/assets/134590095/12bb35b1-e8f7-40cb-8df6-1e597b11196f)

**Comment by jlplenio** [comment]
This is a tunnel connection error. This means that there is a problem with your proxies or internet connection.
Please see if it works with an empty proxy_list.txt and report back.

**Reply by ChickenFlavor** [reply, marked as answer]
Yes it works perfectly fine with no proxies. The proxies that I was using were expired
